.travis.yml

```diff
@@ -49,7 +49,7 @@ before_install:
   - source ./ci/travis/setup.sh
 
 install:
-  - pip install setuptools-rust pytest pytest-benchmark tox
+  - pip install setuptools-rust pytest pytest-benchmark tox tox-venv
 
 script:
   - ./ci/travis/test.sh
```

Comment:
if I understood correctly we can add `virtualenv` support for Linux by installing `tox-venv` and running  script as a separate step like
```yml
install:
  - pip install setuptools-rust pytest pytest-benchmark tox

script:
  - ./ci/travis/test.sh
  - pip install tox-venv
  - ./ci/travis/test.sh
```
but it will increase build time